example/provider-extensions/garden/cloud-profiles/create-cloud-profiles.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
```

Comment:
This file talks about `CloudProfile`s, however it's actually about `ControllerRegistration`s

Reply:
It is creating both 😄 
I'll split the files.